src/sentry/static/sentry/app/views/projectFilters.jsx

```diff
@@ -584,6 +635,42 @@ const ProjectFilters = React.createClass({
     }
   },
 
+  timeLabelAsDay(point) {
+    let timeMoment = moment(point.x * 1000);
+
+    return timeMoment.format('LL');
+  },
+
+  renderTooltip(point, pointIdx, chart) {
+    let timeLabel = this.timeLabelAsDay(point);
+    let totalY = 0;
+    for (let i = 0; i < point.y.length; i++) {
+      totalY += point.y[i];
+    }
+    let {formattedData} = this.state;
+
+    return ReactDOMServer.renderToStaticMarkup(
+      <div style={{width: '150px'}}>
+        <div className="time-label"><span>{timeLabel}</span></div>
+        <div>{intcomma(totalY)} {t('total event')}{totalY !== 1 ? 's' : ''}</div>
```

Comment:
`{t('total event')}{totalY !== 1 ? 's' : ''}`

^Heads up, this won't do translation of a pluralization correctly, because translators will just see "total event" and not realize that we're just stuffing an `s` on the end, regardless of language.

Better to do:

```javascript
{totalY > 1 ? t('total events') : t('total event')}
```

Reply:
👍